### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Aaron-Mandela-Okullo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Header</title>
+</head>
+<body>
+  <div id="root"></div>
+  <script type="module" src="/src/main.jsx"></script>
+  <script  src="https://cdn.tailwindss.com"></script>
```
Comment (fatuma65) [inline index.html:11]: Since we have tailwind installed as a dependency, adding it as a cdn would not be great choice for production. It is not required.

**Review by fatuma65 — COMMENTED** [review]
In the App.jsx file, we are registering our routes, It would be best if you created another component that handles the theme states and leave the app.jsx for routes, please update the application on your machine with the changes already on the main branch.

The header.jsx doesnt have the whole design as implemented in the figma file, especially the search functionality. The logo should be seen as an h2 then a span having the primary color from the design.

File: index.html (hunk 1 above)
Comment (FREDVUNI) [reply index.html:11]: @Aaron-Mandela-Okullo, did you catch this?

**Comment by Aaron-Mandela-Okullo** [comment]
I had actually changed the header

**Comment by FREDVUNI** [comment]
@fatuma65, look into this

**Comment by FREDVUNI** [comment]
we need to close and merge this PR today

cc: @Aaron-Mandela-Okullo and @fatuma65

**Comment by Aaron-Mandela-Okullo** [comment]
Will get on a call with Cathy today at 6 pm and we get that done

**Comment by FREDVUNI** [comment]
Thank you @Aaron-Mandela-Okullo

**Comment by fatuma65** [comment]
Code looks good

**Comment by Aaron-Mandela-Okullo** [comment]
Awesome, thanks!

**Comment by FREDVUNI** [comment]
Alright! Great stuff @Aaron-Mandela-Okullo and @fatuma65

**Comment by Aaron-Mandela-Okullo** [comment]
Thanks for the feedback!

**Pull request merged**